dashboard/src/scss/components/_plugin-card.scss

```diff
@@ -36,6 +47,10 @@
       display: flex;
       align-items: center;
     }
+
+    .components-external-link{
```

Comment:
@cristian-ungureanu I don't think we need this extra style here, we can just add the `plugin-action` class to the component, it has the style already: https://github.com/Codeinwp/themeisle-companion/blob/f09a67da20dcaa95d5337ad8553d73c7094848cf/dashboard/src/scss/components/_plugin-card.scss#L69

Reply:
Nice catch! Can you create another issue, please? I already released it and we'll forget to update next time.